ledger/accountdb.go

```diff
@@ -554,6 +629,50 @@ func (qs *accountsDbQueries) close() {
 	}
 }
 
+// accountsOnlineTop returns the top n online accounts starting at position offset
+// (that is, the top offset'th account through the top offset+n-1'th account).
+//
+// The accounts are sorted by their normalized balance and address.  The normalized
+// address has to do with the reward parts of online account balances.  See the
+// normalization procedure in onlineAccount.normBalance().
+//
+// Note that this does not check if the accounts have a vote key valid for any
+// particular round (past, present, or future).
+func accountsOnlineTop(tx *sql.Tx, offset, n uint64) (map[basics.Address]*onlineAccount, error) {
+	rows, err := tx.Query("SELECT address, data FROM onlineaccountbase ORDER BY normalizedbalance DESC, address DESC LIMIT ? OFFSET ?", n, offset)
```

Comment:
the `accountsOnlineTop` function is only called in a loop, always starting from offset 0.
In that case, why do we need the secondary sort by address ?
i.e. if we need to ensure stable order across calls, it would suffice to use _rowid_ which would allow you to simplify the index

Reply:
Because the sort order must be exactly the same across all computers running algod

Reply:
ahh, because you scanning only a subset of the online accounts. Make sense.